lms/djangoapps/faq/views.py

```diff
@@ -0,0 +1,49 @@
+import re
+
+from django.http import JsonResponse
+from django.utils.safestring import mark_safe
+from django.views.decorators.csrf import csrf_exempt
+from edxmako.shortcuts import render_to_response, render_to_string
+
+from lms.djangoapps.faq.models import Faq
+
+
+def get_faq(request):
+    """
+    Display the Dynamic FAQ Page
+    """
+
+    faq_page = Faq.objects.filter(is_active=True).last()
+
+    if not faq_page:
+        context = {
+            'title': 'FAQ',
+            'body': '** Please add content for FAQ page',
+        }
+    else:
+        body = re.search('<body>(.*)</body>', faq_page.content, re.S)
```

Comment:
why we are searching in body tags? why not strip them off if they exist?

Reply:
We have a source of a complete html page at backend we just need to extract the body from it.